README.md

```diff
@@ -48,7 +48,12 @@ Depending on your operating system, you will need to install:
 Install the current [version of Python](https://devguide.python.org/versions/) from the
 [Microsoft Store](https://apps.microsoft.com/store/search?publisher=Python+Software+Foundation).
 
-Install tools and configuration manually:
+Install tools with [Chocolatey](https://chocolatey.org):
+``` bash
+choco install python visualstudio2022-workload-vctools -y
```

Comment:
Since this installs Python and VS together, should it maybe go on top of this section, above instructions to install Python?

Also, in that case, adjust a comment to say `Or install and configure Python and Visual Studio tools manually`.

Reply:
Moved the chocolatey bash to the top
https://github.com/nodejs/node-gyp/pull/3008/commits/abe92020e5151ddd9172eef393bce32dc3873b41

I also think it now makes sense to have the python manual installation guide be moved one level down like the one after it
https://github.com/nodejs/node-gyp/pull/3008/commits/9bc8fa50785435546073b78ba9ce96f5473226c1